### ISSUE TITLE
Problem downloading Client

### ISSUE THREAD

**Issue by Jinz33** [kzahel/jstorrent-legacy-app#177]
after powerwashing my chromebook and updating to newest version of chrome, the web store is making me purchase again.
I have my transaction ID if needed.

**Comment by kzahel** [comment]
Did you sign in with the same account? You can send your order id to support@jstorrent.com if you want a refund

**Comment by Jinz33** [comment]
Yes I did. The card used to purchase it is no longer active so refund can
not be to back on it. Is there a way I can get a direct download
link. 10370910874874185126.0fc2abd191d14d0499b7e46654404278
He is my purchase id

**Comment by kzahel** [comment]
Sorry not much I can do in that case! You can use the free version (search for JSTorrent Lite on google) and uninstall/reinstall when you run out of downloads.

**Comment by Jinz33** [comment]
Ok thanks

**Comment by kzahel** [comment]
The new JSTorrent is completely free — no purchases needed. Get it at **[jstorrent.com](https://jstorrent.com)**

If you run into any issues, please leave feedback!
